jumpscale/packages/tfgrid_solutions/chats/publisher.py

```diff
@@ -80,11 +80,11 @@ def publisher_info(self):
     @chatflow_step(title="Solution Settings")
     def configuration(self):
         form = self.new_form()
-        ttype = form.single_choice("Choose the type", options=["wiki", "www", "blog"], default="wiki", required=True)
+        ttype = form.single_choice("Choose Your Publicattion Type", options=["Wiki", "www", "Blog"], default="Wiki", required=True)
```

Comment:
this result will be affecting a lot of checks, we shouldn't change that

Reply:
if its a multiple choice, a beginning of a sentence, please always ALWAYS big first letter

Blog
Wiki
Website (dont call it www)